run.sh

```diff
@@ -35,10 +35,12 @@ prepare () {
   # These directiories essentially define the structure of our API endpoints.
   # TODO: These should perhaps live in python, near the schemas (defined in api/)?
   STATES_DIR="${API_OUTPUT_DIR}/";
+  INPUT_BASE_DIR="${API_OUTPUT_DIR}/";
```

Comment:
We have API_OUTPUT_DIR, STATES_DIR, and INPUT_BASE_DIR that all point at the same thing, which seems confusing.

I actually screwed up below (line 57) and used the wrong one.

I vote we don't add another one and you just use API_OUTPUT_DIR or STATES_DIR.

Reply:
ya i really struggled with this file LOL let me see i think that the API docs had state and county not the plurals